src/components/main/main.js

```diff
@@ -110,6 +111,10 @@ app.component('main', {
             delete res.publicKey;
           }
 
+          this.notify.about(
```

Comment:
Please check if `res.balance > this.account.get().balance` then call this method.